package.json

```diff
@@ -62,6 +62,7 @@
     "react-native-lottie-splash-screen": "^1.0.1",
     "react-native-mail": "^6.1.1",
     "react-native-navigation": "^7.27.1",
+    "react-native-orientation-locker": "^1.5.0",
```

Comment:
Please can you explain why this library is needed. Can we just use navigation option for landscape mode

Reply:
https://wix.github.io/react-native-navigation/docs/style-orientation

Reply:
@Parveshdhull  This library is needed to listen to device orientation changes. Navigation option is used to change the orientation.

Reply:
i see we make changes in the native code, is it possible to avoid using this library and just use status native code ?

Reply:
@flexsurfer I am not sure what do you mean by "status native code"

Reply:
I guess something like,
https://stackoverflow.com/questions/5726657/how-to-detect-orientation-change-in-layout-in-android
https://stackoverflow.com/questions/38894031/swift-how-to-detect-orientation-changes

Reply:
Oh okay thanks @Parveshdhull 

@flexsurfer sure I can implement the methods natively

Reply:
@flexsurfer I was investigating what the added native code lines are used for, but they seem not to be needed to listen to device orientation changes. I have removed them. So is it okay to keep using this library, as I would prefer to use its hooks?